phsite/settings.py

```diff
@@ -12,11 +13,13 @@
 
 from pathlib import Path
 import os
-try: 
-    from config import DJANGO_SECRET, POSTGRES_PASSWORD
-except:
-    DJANGO_SECRET = os.environ.get('DJANGO_SECRET')
-    POSTGRES_PASSWORD = os.environ.get('POSTGRES_PASSWORD')
+
+DJANGO_SECRET = os.environ.get('DJANGO_SECRET')
+POSTGRES_PASSWORD = os.environ.get('POSTGRES_PASSWORD')
+
+# check that values are not None
+assert DJANGO_SECRET 
+assert POSTGRES_PASSWORD
```

Comment:
Ensure the values for the password and secret actually are present in `settings.py`